Source/WebCore/platform/graphics/FormatConverter.cpp

```diff
@@ -251,12 +251,12 @@ template<> ALWAYS_INLINE void unpack<GraphicsContextGL::DataFormat::BGRA8, uint8
     }
 }
 
-template<> ALWAYS_INLINE void unpack<GraphicsContextGL::DataFormat::RGBA5551, uint16_t, uint8_t>(const uint16_t* source, uint8_t* destination, unsigned pixelsPerRow)
+template<> ALWAYS_INLINE void unpack<GraphicsContextGL::DataFormat::RGBA5551, uint16_t, uint8_t>(std::span<const uint16_t> source, std::span<uint8_t> destination, unsigned pixelsPerRow)
 {
 #if HAVE(ARM_NEON_INTRINSICS)
-    SIMD::unpackOneRowOfRGBA5551ToRGBA8(source, destination, pixelsPerRow);
+    SIMD::unpackOneRowOfRGBA5551ToRGBA8(source.data(), destination.data(), pixelsPerRow);
```

Comment:
This breaks the arm32 build as the SIMD method expects a reference to a pointer:

```
ALWAYS_INLINE void unpackOneRowOfRGBA5551ToRGBA8(const uint16_t*& source, uint8_t*& destination, unsigned& pixelsPerRow)
```
https://build.webkit.org/#/builders/893/builds/18641/steps/9/logs/errors

There are other similar errors due to changes like this one.